app/code/Magento/CatalogGraphQl/Model/Resolver/Products/Query/Search.php

```diff
@@ -148,4 +159,27 @@ private function buildSearchCriteria(array $args, ResolveInfo $info): SearchCrit
 
         return $searchCriteria;
     }
+
+    /**
+     * Get product attribute value
+     *
+     * @param Product $product
+     * @param array $productArray
+     * @param string $field
+     *
+     * @return string|null|int|bool
+     */
+    private function getAttributeValue(Product $product, array $productArray, string $field)
+    {
+        if ($attribute = $product->getResource()->getAttribute($field)) {
+            $attributeValue =  $attribute->getFrontend()->getValue($product);
+            if ($attributeValue && !($attributeValue instanceof Phrase)) {
+                return $attributeValue;
```

Comment:
is this how we translate the label? or what does $attribute->getFrontend()->getValue($product) do?
it's best to inject frontend class ..whatever that class is...
Hope is not luma related..